src/components/Pagination/Pagination.module.scss

```
@@ -0,0 +1,70 @@
+﻿@import "../../styles/mixins";
+
+
+.pagination {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.page {
+  @include charlie();
+  color: var(--brand-aubergine);
+  height: 3.2rem;
+  width: 3.2rem;
+  cursor: pointer;
+  font-weight: var(--font-weight-bold);
+  border: 0;
+  border-radius: 1.6rem;
+  margin: 0 var(--spacer-three-xs);
+  transition: background 200ms ease-out, padding 0.2s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+
```

Please change the background to be transparent for pagination buttons.

```suggestion
  background: transparent;
```